### DISCUSSION TITLE
mistake

### DISCUSSION THREAD

**Discussion by zwdcoder** [mifi/editly#269] [Q&A]
author , please tell me,when executing the following command I get this error:
editly, /home/JobId_1.json5, --width, 900, --height, 506

 Error: gl returned null, this probably means that some dependencies are not installed. See README.
    at default (file:///root/.nvm/versions/node/v18.16.1/lib/node_modules/editly/glTransitions.js:14:11)
    at Editly (file:///root/.nvm/versions/node/v18.16.1/lib/node_modules/editly/index.js:171:60)
    at async file:///root/.nvm/versions/node/v18.16.1/lib/node_modules/editly/cli.js:140:3

**Comment by bkeepers** [comment]
Are you running on a headless linux instance? Make sure xvfb is installed and running. See https://github.com/stackgl/headless-gl?tab=readme-ov-file#how-can-headless-gl-be-used-on-a-headless-linux-machine
